Tests/IntegrationTests/Components Tests/Gift Card/GiftCardComponentTests.swift

```diff
@@ -115,14 +114,49 @@ class GiftCardComponentTests: XCTestCase {
         
         // When
         setupRootViewController(sut.viewController)
-        wait(for: .milliseconds(300))
 
         // Then
         XCTAssertNotNil(expiryDateItemView, "should have expiry date field for meal voucher")
         XCTAssertNotNil(securityCodeItemView, "security code should be shown by default")
         XCTAssertEqual(securityCodeItemTitleLabel?.text, "Security code", "cvc title changes based on payment method")
     }
 
+    func testMealVoucherDetails() {
+
+        // Given
+        let paymentMethod = MealVoucherPaymentMethod(type: .mealVoucherSodexo, name: "Sodexo")
+        sut = GiftCardComponent(
+            partialPaymentMethodType: .mealVoucher(paymentMethod),
+            context: context,
+            amount: amountToPay,
+            publicKeyProvider: publicKeyProvider
+        )
+
+        // When
+        setupRootViewController(sut.viewController)
```

Comment:
Could you please try instantiating just view controller and calling it's `loadViewIfNeeded()` instead? This would significantly decrease test run and it would be more an integration not a UI test anymore.
[Here is](https://github.com/Adyen/adyen-ios/blob/develop/Tests/IntegrationTests/Actions%20Tests/Await/AwaitComponentTests.swift#L120) an example refactor I've done before to reduce overall tests run time.

It wouldn't work for some tests that requite `window` to be present but for 90% of all tests it would work.

Reply:
Fix [a97b509](https://github.com/Adyen/adyen-ios/pull/1904/commits/a97b509b22558557f5f663919ef6df5c776ba131)

Reply:
Thank you!